### PR TITLE
fix the Dashboard tab title

### LINKED ISSUE #18581: Fix the Dashboard next tab title
### Is your task related to a problem? Please describe.
Currently, dashboard next browser tab has title: New-dashboard.
On the existing Dashboard it's `Project(Product) Name | Tab name`, like `Eclipse Che | Get Started`.
It should be the same on Dashboard Next

### PR DESCRIPTION
**Pull request by olexii4**
### What does this PR do?
Fix the Dashboard tab title.

### Which issue does this PR related to?
fixed https://github.com/eclipse/che/issues/18581

Signed-off-by: Oleksii Orel <oorel@redhat.com>
Screenshot from 2020-12-23 17-50-20

### CODE REVIEW

File: src/pages/WorkspaceDetails/index.tsx [hunk 1]
```diff
@@ -153,6 +154,7 @@ export class WorkspaceDetails extends React.PureComponent<Props, State> {
 
     return (
       <React.Fragment>
+        <Head pageName="Workspace Details" />
```
Comment (sleshchenko) [inline src/pages/WorkspaceDetails/index.tsx]: As we currently do, it's better to use workspace name here.

File: src/pages/IdeLoader/index.tsx [hunk 2]
```diff
@@ -295,6 +296,7 @@ class IdeLoader extends React.PureComponent<Props, State> {
 
     return (
       <React.Fragment>
+        <Head pageName="Workspace Loader" />
```
Comment (sleshchenko) [inline src/pages/IdeLoader/index.tsx]: I don't have a strong opinion here but maybe something like `Loading Workspace` or even `Loading WSNAME` would be better, currently, we have just `workspace name` which is good as well.

**Review by sleshchenko — COMMENTED** [review]
When IDE is opened, the title is just Eclipse Che, but it would be better to show workspace name, currently, we have `Eclipse Che | $workspacename`

File: src/pages/IdeLoader/index.tsx (hunk 2 above)
Comment (olexii4) [reply src/pages/IdeLoader/index.tsx]: I have fixed it

File: src/pages/WorkspaceDetails/index.tsx (hunk 1 above)
Comment (olexii4) [reply src/pages/WorkspaceDetails/index.tsx]: I have fixed it.

**Comment by olexii4** [comment]
Screenshot from 2020-12-24 14-48-41

Screenshot from 2020-12-24 14-48-23

**Review by sleshchenko — APPROVED** [review]
Seems to work as expected for me

**Review by akurinnoy — APPROVED** [review]

**Pull request merged**